lib/src/timer/timer.component.ts

```diff
@@ -0,0 +1,64 @@
+import {
+    Component,
+    Input,
+    ViewEncapsulation,
+    Output,
+    EventEmitter,
+    OnInit,
+    OnDestroy,
+    OnChanges,
+    SimpleChanges,
+    ChangeDetectorRef,
+} from "@angular/core";
+import { timer, Observable } from "rxjs";
+import { map, takeWhile, tap, finalize } from "rxjs/operators";
+
+@Component({
+    selector: "sebng-timer",
+    templateUrl: "./timer.component.html",
+    encapsulation: ViewEncapsulation.None,
+})
+export class TimerComponent implements OnInit, OnChanges {
+    @Input() id?: string;
+    @Input() className?: string;
+    @Output() callback = new EventEmitter<void>();
```

Comment:
please call this something more meaningful like onTimerEnd

Reply:
Yah, make sense